big_tests/dynamic_domains.config

```diff
@@ -38,5 +56,121 @@
         {username, <<"bOb">>},
         {server, <<"domain.example.com">>},
         {host, <<"localhost">>},
-        {password, <<"makrolika">>}]}
+        {password, <<"makrolika">>}]},
+    {carol, [
```

Comment:
Please only add users that are actually used by the enabled test suites.

Reply:
done